src/main/java/com/aws/greengrass/util/platforms/windows/WindowsExec.java

```diff
@@ -140,6 +144,62 @@ public synchronized void close() throws IOException {
             return;
         }
 
+        try {
+            stopGracefully();
+        } finally {
+            stopForcefully();
+        }
+    }
+
+    private void stopGracefully() {
```

Comment:
break this out into a method which accepts a pid as input. In Tilo's change we'll know the pid